core/Dockerfile

```diff
@@ -23,5 +29,8 @@ COPY services /home/pi/services
 COPY start-companion-core /usr/bin/start-companion-core
 RUN /home/pi/services/install-services.sh
 
+# Copy frontend built on frontendBuild to this stage
```

Comment:
why `frontendBuild` is in camel case ?

Reply:
No specif reason. Any suggestion?

Reply:
@Williangalvani do you know if there's any limitation on the stage's name or a name convention?

Reply:
'Copy multistage frontend output to docker'

Reply:
Ok, but any suggestion on the name of the stage?

Reply:
that is probably on me. The multi-stage examples only have 1-word names, and they are all lower-case. `frontendbuild` gets hard to read, so I made it camelCase.

Reply:
Ha, I just saw that this is the name of the multistage